.github/workflows/pr-to-update-choco.yml

```diff
@@ -0,0 +1,116 @@
+name: PR for updating Chocolatey
+
+# This action will run after a tag starting with "v" is published
+on:
+  push:
+    tags:
+      - 'v*'
+  workflow_dispatch:
+
+permissions:
+  contents: read
+
+jobs:
+  choco-update:
+    permissions:
+      contents: write # for Git to git push
+      pull-requests: write # for creating PRs
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Headlamp
+        uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4.1.7
+        with:
+          token: ${{ secrets.KINVOLK_REPOS_TOKEN }}
+          # we need the full history for the git tag command, so fetch all the branches
+          fetch-depth: 0
+      - name: Configure Git
+        run: |
+          user=${{github.actor}}
+          if [ -z $user ]; then
+            user=vyncent-t
+          fi
+          git config --global user.name "$user"
+          git config --global user.email "$user@users.noreply.github.com"
+      # Set up Node.js environment, pay attention to the version
+      # Some features might not be available in older versions
+      - name: Create node.js environment
+        uses: actions/setup-node@60edb5dd545a775178f52524783378180af0d1f8 # v4.0.2
+        with:
+          node-version: '21'
+      # Install the app dependencies for the choco script
+      - name: Install app dependencies
+        run: |
+          cd $GITHUB_WORKSPACE/app
+          npm ci
+      # We set the latest tag as an environment variable before we use it in the next steps
+      # note that we have to echo the variable to the environment file to make it available in the next steps
+      - name: Set latest tag
+        run: |
+          echo "Setting latest tag"
+          latestTag=$(git tag --list --sort=version:refname 'v*' | tail -1)
+          # Remove the 'v' from the tag
+          latestTag=${latestTag#v}
+          echo "LATEST_HEADLAMP_TAG=$latestTag" >> $GITHUB_ENV
+          echo $latestTag
+      - name: Download checksum
+        run: |
+          echo "Downloading checksum"
+          cd $GITHUB_WORKSPACE/app/windows/chocolatey
+          curl -L -O "https://github.com/headlamp-k8s/headlamp/releases/download/v$LATEST_HEADLAMP_TAG/checksums.txt"
```

Comment:
Maybe we should download this into a different directory (maybe in /tmp), so we don't end up at some point adding it by mistake. As this PR does.